transactions/src/test/java/org/jboss/hal/testsuite/test/transactions/TransactionCommitAndRollbackTestCase.java

```diff
@@ -102,18 +106,39 @@ public void before() {
             "AAAAAAAAAAAAAP__CigFZtXWO0pXxVwMAAAFsgAAAAEAAAAAAAAAAAAAAAAAAP__CigFZtXWO0pXxVwMAAAFrTEHAgIA"
     };
 
-    private static final Vector<PreparedTransactionTestCell> TRANSACTIONS_TEST_DATA = new Vector<PreparedTransactionTestCell>() {
+    private static final Vector<PreparedTransactionTestCell> GUI_TRANSACTIONS_TEST_DATA = new Vector<PreparedTransactionTestCell>() {
         {
         add(new PreparedTransactionTestCell(TRANSACTIONS_XID[0], PerformedAction.ROLLBACK));
         add(new PreparedTransactionTestCell(TRANSACTIONS_XID[1], PerformedAction.COMMIT));
         add(new PreparedTransactionTestCell(TRANSACTIONS_XID[2], PerformedAction.COMMIT));
         add(new PreparedTransactionTestCell(TRANSACTIONS_XID[3], PerformedAction.ROLLBACK));
         add(new PreparedTransactionTestCell(TRANSACTIONS_XID[4], PerformedAction.COMMIT));
-        add(new PreparedTransactionTestCell(TRANSACTIONS_XID[5], PerformedAction.ROLLBACK));
-        add(new PreparedTransactionTestCell(TRANSACTIONS_XID[6], PerformedAction.COMMIT));
         }
     };
 
+    private static final int[] REFRESH_TEST_INDEXES = {5, 6};
```

Comment:
Actually I don't understand the need for `REFRESH_TEST_INDEXES` constant array. IMHO this is just making code less readable. You don't have something like `GUI_TEST_INDEXES` (and not having such variable is ok IMO)